### ISSUE TITLE
test-data-bot seems to be outdated

### ISSUE THREAD

**Issue by PowerSlime** [kentcdodds/react-testing-library-course#29]
https://github.com/kentcdodds/react-testing-library-course/blob/88bbc8bc2465a436c8197717dfd673211bc36e90/src/__tests__/tdd-06-generate-data.js#L3

There is already newer version of `test-data-bot` https://www.npmjs.com/package/@jackfranklin/test-data-bot.

Looks like there not so much breaking changes was introduced, and it's even written in TypeScript!

**Comment by kentcdodds** [comment]
Yep! I want to keep the code in this repo consistent with the videos so I'm not going to do any code updates until the videos can be updated.

Thanks!
